CIP-0021/CIP-0021.md

```diff
@@ -71,6 +79,20 @@ It is allowed to arbitrarily combine other supported certificate types, but all
 
 Since withdrawals are represented as a map of reward accounts, withdrawals also need to be sorted in accordance with the specified canonical CBOR format. A transaction must not contain duplicate withdrawals. All withdrawals included in a transaction must have the same type of stake credential i.e. either 0 - `addr_keyhash` or 1 - `scripthash`. The stake credential type must also be consistent with the type used for certificates.
 
+**Auxiliary data**
+
+Auxiliary must be encoded in their "tuple" format i.e.:
+
+```
+[ transaction_metadata: { * transaction_metadatum_label => transaction_metadatum }, auxiliary_scripts: [ * native_script ]]
+```
+
+instead of:
+
+```
+{ * transaction_metadatum_label => transaction_metadatum }
+```
```

Comment:
Let's replace this "instead of" part with
"The item `auxiliary_scripts` must be an array of length 0."

Reply:
https://github.com/vacuumlabs/CIPs/pull/4/commits/707b4f37e97c2b4287930bc49efa53ec864c9ac7